generate_html.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
+
+import duckdb
+import pandas as pd
+from jinja2 import Environment, FileSystemLoader
+
+TEMPLATES_PATH = "templates"
+INDEX_TEMPLATE = "index.html"
+
+
+def main():
+    # Load data from github_contributions.duckdb into pandas dataframe
+    con = duckdb.connect(database="github_contributions.duckdb")
+    df = pd.read_sql_query("SELECT * FROM src_pull_requests", con)
```

Comment:
I think we should add consumer models that contains the right information and structure
- Filter for Xebia organization

1. Build a model with top PRs
2. Build a model with unique contributors per project and/or owner

etc.